index.rst

```diff
@@ -890,27 +860,49 @@ More details to follow.
 Telescope Cabinet Switchable PDU
 --------------------------------
 
-In the event that a controller in the cabinet needs power cycling remotely, this may be done by logging into the switchable PDU mounted in the cabinet. The IP and connection info can be found `here <https://confluence.lsstcorp.org/x/qw6SBg>`_
+.. warning::
+    DO NOT TRY THIS.
+    This PDU server is where the camera ion pump and other critical components
```

Comment:
This is no longer true :-) But i can fix this in another ticket when i make the other updates.